boa_engine/src/builtins/intl/mod.rs

```diff
@@ -29,13 +33,23 @@ impl BuiltIn for Intl {
         let _timer = Profiler::global().start_event(Self::NAME, "init");
 
         let string_tag = WellKnownSymbols::to_string_tag();
+        let date_time_format = FunctionBuilder::native(context, DateTimeFormat::constructor)
+            .name("Intl.DateTimeFormat")
```

Comment:
```suggestion
            .name("Intl.DateTimeFormat")
```
The name should be `"DateTimeFormat"`

Reply:
Done https://github.com/boa-dev/boa/pull/2025/files#diff-1dd4af22d3a6142a8060d056e9d0489d84d8a4d9c68a8f6e7b359b8d3ab35499R37

Do I need to change the name [here](https://github.com/boa-dev/boa/pull/2025/files#diff-d3b068b8d1eb849b10fde7458b9cfc96c68353420893ef2ca613b258884c1756R49) as well?

Reply:
> Do I need to change the name [here](https://github.com/boa-dev/boa/pull/2025/files#diff-d3b068b8d1eb849b10fde7458b9cfc96c68353420893ef2ca613b258884c1756R49) as well?

Yes, it should be the name of the constructor